### PR TITLE
Add option for the dependent jobs run

### PR DESCRIPTION
**Pull request by jankohoutek**
Add the option to run the jobs also with dependencies

### CODE REVIEW

File: openqa-trigger-bisect-jobs [hunk 1]
```diff
@@ -44,7 +46,9 @@ def call(cmds, dry_run=False):
     return subprocess.call((['echo', 'Simulating: '] if dry_run else []) + cmds)
 
 
-def openqa_clone(cmds, dry_run, default_opts=['--skip-chained-deps', '--within-instance'], default_cmds=['_GROUP=0']):
+def openqa_clone(cmds, dry_run, include_depends, default_opts=['--skip-chained-deps', '--within-instance'], default_cmds=['_GROUP=0']):
+    if include_depends == True:
+      default_opts = ['--within-instance','--parental-inheritance']
```
Comment (okurz) [inline openqa-trigger-bisect-jobs:51]:
```suggestion
    if include_depends:
      default_opts = ['--within-instance','--parental-inheritance']
```

File: openqa-trigger-bisect-jobs [hunk 2]
```diff
@@ -26,6 +26,8 @@ def parse_args():
                         help='The openQA test URL for which to trigger bisection investigation jobs')
     parser.add_argument('--dry-run', action='store_true',
                         help='Do not do any action on openQA')
+    parser.add_argument('--include-depends', action='store_true', default=False,
+                        help='Restart also the jobs on which investigated one is dependant')
```
Comment (okurz) [inline openqa-trigger-bisect-jobs:30]:
```suggestion
    parser.add_argument('--include-depends', action='store_true', default=False,
                        help='Restart also the jobs on which the investigated one depends')
```

**Review by okurz — CHANGES_REQUESTED** [review]
please phrase the git commit message according to https://github.com/os-autoinst/scripts#rules-for-commits , i.e. with imperative mood

**Review by okurz — COMMENTED** [review]
The option certainly does what you want but the idea for "openqa-trigger-bisect-jobs" was that it does what is needed to do to bisect any regression. And how should the caller decide if they need the option "--include-depends" or not? The problem that motivated you was a package change that affected the parent installation job already, right? I am wondering, if we would simply always do that, what would be the drawbacks?

**Comment by kalikiana** [comment]
> The option certainly does what you want but the idea for "openqa-trigger-bisect-jobs" was that it does what is needed to do to bisect any regression. And how should the caller decide if they need the option "--include-depends" or not? The problem that motivated you was a package change that affected the parent installation job already, right? I am wondering, if we would simply always do that, what would be the drawbacks?

This doesn't seem to be linked to a ticket or issue but [#95783](https://progress.opensuse.org/issues/95783#note-41) touches on very similar points.

**Comment by okurz** [comment]
closing due to no response. At any point feel free to address the open points and re-open.

**Pull request closed without merging**